### ISSUE TITLE
Flexible Notifications settings inside loop app

### ISSUE THREAD

**Issue by Chester-sk8** [LoopKit/Loop#2250]
Good day. 

Is it possible to implement some notifications settings inside the loop?
It would be great if user can turn off or postpone some alerts like  about empty reservoir or about small amount of insulin in the reservoir. 
It would be great for medtronic’s pumps users because if pump shows you 0 unit, in reality there are about 20 units inside and loop app just starts to be quite annoying you know. 

But I don’t want to turn off totally all notifications from loop for case when it lose signal and etc. 

In other words it would be great to have some settings for notifications like it is on xdrip4ios for example. 

Thank you.
